lib/sort_by_length.rb

```diff
@@ -1,7 +1,21 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: o(n)
+# Space complexity: 0(n)
 def sort_by_length(my_sentence)
```

Comment:
This works but the time complexity is O(n^2) as you can't sort, generally with O(n) time complexity. Otherwise this is a good insertion sort.